build.gradle

```diff
@@ -0,0 +1,32 @@
+plugins {
+    id 'java-library'
+}
+
+group 'org.example'
+version '1.3.5'
+
+targetCompatibility = 8
+sourceCompatibility = 8
+
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.0'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.0'
+    implementation 'org.apache.commons:commons-lang3:3.10'
```

Comment:
We should put dependencies for the main source set first and then dependencies for the test source set.
So `implementation` dependencies should be located before `testImplementation` and `testRuntimeOnly` dependencies.

Reply:
So as I got first we write dependencies for the implementation and after that for the tests. Reordered it